scripts/start-https.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
```

Comment:
```suggestion
const fs = require('node:fs');
const path = require('node:path');
const { execSync } = require('node:child_process');
```

nit: using the `node:` prefix makes clearer that we're using built-in node modules.